util/data.go

```diff
@@ -21,6 +21,7 @@ var (
 	}
 
 	ErrInvalidConceptTypeFormat              = "invalid concept type %v"
+	ErrMaxIdsLimitFormat                     = "invalid 'ids' parameter size supplied: %v; the max size of 'ids' is %v"
```

Comment:
The description of error is confusing for me. I'd suggest something like:
```suggestion
	ErrMaxIdsLimitFormat                     = "number of 'ids' parameters exceeds the limit, supplied: %v; the max number of 'ids' is %v"
```

Reply:
Agree